src/reducers/entityLoadingReducer.js

```diff
@@ -10,7 +10,7 @@ const initialState = {
 // item that we are having problems with getting its info, ie getCluster or getNodePool
 const entityLoadingReducer = produce((draft, action) => {
   const { type, id } = action;
-  const matches = /(.*)_(REQUEST|SUCCESS|ERROR|FINISHED|NOT_FOUND)/.exec(type);
+  const matches = /(.*)_(REQUEST|SUCCESS|ERROR|FINISHED|NOT_FOUND)$/.exec(type);
```

Comment:
To make sure we don't thing `SINGLE_ERROR_CLEAR` is a request, I added this `$` so that only actions ending in one of these words are considered for the loading reducer.